thirdeye-ui/src/app/components/impact-dashboard/detection-performance/data/index.tsx

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright 2024 StarTree Inc
+ *
+ * Licensed under the StarTree Community License (the "License"); you may not use
+ * this file except in compliance with the License. You may obtain a copy of the
+ * License at http://www.startree.ai/legal/startree-community-license
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the
+ * License is distributed on an "AS IS" BASIS, WITHOUT * WARRANTIES OF ANY KIND,
+ * either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+import { useEffect, useState } from "react";
+import { DataProps, DetectionData } from "./interfaces";
+
+export const useGetData = ({
```

Comment:
generic variable name

Reply:
@nalin-patidar Can we have a specific name for these functions? If you want to do it in a separate PR, that's also fine.